plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -370,6 +370,50 @@ fu_plugin_uefi_capsule_update_splash (FuPlugin *plugin, FuDevice *device, GError
 	return fu_plugin_uefi_capsule_write_splash_data (plugin, device, image_bmp, error);
 }
 
+static gboolean
+fu_capsule_update_on_disk (FuDevice *device, GError **error)
+{
+	g_autofree guint64 *os_indications_supported = NULL;
+	g_autofree guint64 *os_indications = NULL;
+	guint32 attr;
+	gsize os_ind_size = 0;
+
+	if (fu_device_get_metadata_boolean (device, "DisableCapsuleUpdateOnDisk"))
```

Comment:
is your idea here that you would set this from a conffile?

Reply:
Yea. Basically this becomes the default, unless you:
1. disable it
2. firmware doesnt support it

But I am mostly looking into feedback here, since this might break existing devices that support capsuleupdate on disk (but that should be really rare).
Alternatively I could look into getting rid of it and try to locate the protocol for runtime capsule update.  If that's not supported we could fallback to the on-disk variant